cm/base/BaseCM/test_hddcdd.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Comment:
These tests are related to the CM hddcdd, ideally they should be in the directory of this CM 
If the tested features are used by several CMs, they will just have to be tested several times (maybe with different data depending on how the CM works)

Reply:
yes it makes sense, we can keep the codes as you wrote it